### PR TITLE
Added m to cm Feature

### PR DESCRIPTION
**Pull request by onlyoneaman**
I added an m-cm Feature in the calculator. It is as per the changes mentioned by you in the previous PR.
I am closing my PR #10 as the next PR #11 is also focussed on cm-m Feature.
My Program is capable of taking an array as input and prints out all numbers converted in cm with space in between them. 
You can have a look and I will be waiting for your review.

### CODE REVIEW

File: cli.py [hunk 1]
```diff
@@ -19,6 +19,8 @@ def calculate(operation, integers):
     return calc.multiply(integers)
   if (operation == 'average'):
     return calc.average(integers)
+  if (operation == 'm-cm'):
+  	return ' '.join(map(lambda x:str(x), calc.m_to_cm(integers)))
```
Comment (itaditya) [inline cli.py]:
```suggestion
  	return '\n'.join(map(lambda x:str(x), calc.m_to_cm(integers)))
```

**Comment by itaditya** [comment]
The code just needs one minor change, after that I'll merge but please don't open other PRs in this repo as I'm saving it for my mentees.

**Comment by itaditya** [comment]
Also always raise a PR from a branch other than master so I can pull the code of your PR locally and see everything is working without corrupting my master branch

**Comment by onlyoneaman** [comment]
> Also always raise a PR from a branch other than master so I can pull the code of your PR locally and see everything is working without corrupting my master branch

I will take care of it from next time anywhere.

**Comment by onlyoneaman** [comment]
> The code just needs one minor change, after that I'll merge but please don't open other PRs in this repo as I'm saving it for my mentees.

OK and Thanks for helping me here

**Pull request merged**